### ISSUE TITLE
Modulus Grid Snapping

### ISSUE THREAD

**Issue by TresAbhi** [Fyrestar/THREE.InfiniteGridHelper#6]
It would be pretty neat if you could make the grid snap to a multiple of a certain number. This behavior can be seen in graphic calculators and would be especially useful in a top-to-down cartographic view with orbit controls.

![](https://i.imgur.com/1zb3hyz.gif)

**Comment by Fyrestar** [comment]
Yes i added this for the update already, but since you mention it, also with an option for orthographic cameras as for 3D the camera to plane distance is used.

**Comment by TresAbhi** [comment]
This is big! I have tried tinkering with the shaders (having no clue what I'm doing) and failed. Good to see this is coming to :)

**Comment by TresAbhi** [comment]
> Likely this or next week

Any updates?

**Comment by tingspain** [comment]
Any update in this feature?  ^___^

**Comment by tresabhi** [comment]
@Fyrestar Do you have anything new to share about this?

**Comment by Fyrestar** [comment]
Sadly people are just stealing/claiming code everywhere (i mean literally just the repos to claim as their own project), so i heavily turned open sourcing down.
